FreeRTOS-Plus/Demo/coreHTTP_Windows_Simulator/HTTP_S3_Download/DemoTasks/S3DownloadHTTPExample.c

```diff
@@ -250,17 +264,6 @@ void vStartSimpleHTTPDemo( void )
 
 /**
  * @brief Entry point of the demo.
- *
```

Comment:
Why delete this?

Reply:
I had been referring to the MQTT multitask demo and saw that there was no description, but I now realize the other demos have included it so I have reverted the change. Thanks.